src/components/Calender.vue

```diff
@@ -15,7 +15,15 @@
 		>
 			<template v-if="!calVal.isPadding">
 				<h3>
-					{{ calVal.date + 1 }}
+					{{
+						"".concat(
+							String(calVal.date + 1) +
+								" " +
+								listToDisplay.get(
+									Math.floor((calVal.date + 1) / 2)
+								)
```

Comment:
This returns an object, of which you want the `name` property. See the `title` attribute set on the link (line 12)

Reply:
Yes just found that thanks